.github/workflows/build.yml

```diff
@@ -48,16 +48,15 @@ jobs:
           # trick.
           echo "GOPATH=$GITHUB_WORKSPACE" >> $GITHUB_ENV
           echo "GOBIN=$GITHUB_WORKSPACE/bin" >> $GITHUB_ENV
-          echo "GO111MODULE=off" >> $GITHUB_ENV # until we solve this for coredhcp-generator
       - name: build coredhcp-generator
         run: |
           set -exu
           cd "${GITHUB_WORKSPACE}"/src/github.com/${{ github.repository }}/cmds/coredhcp-generator
-          GO111MODULE=on go build
+          go build
           builddir=$(./coredhcp-generator -f core-plugins.txt)
           cd "${builddir}"
           ls -l
-          go get -v ./...
+          go mod init "coredhcp"
```

Comment:
this is OK for the CI - otherwise the correct thing would be "go mod init github.com/coredhcp/coredhcp". I wonder if doing otherwise might break anything in the future when we will move to modules

Reply:
actually `go mod init github.com/coredhcp/coredhcp` breaks it, because it then expects all the imports from `github.com/coredhcp/coredhcp` to be in the current directory. But this directory only has the main entrypoint which is normally in `github.com/coredhcp/coredhcp/cmds/coredhcp`, so it doesn't manage to import anything

to be able to use `go mod init github.com/coredhcp/coredhcp` we'd need to build the generated `coredhcp.go` in-place inside the source directory, which is more painful
I could maybe rename the module as "local-coredhcp-entrypoint" or something more explicit like that ?

Reply:
ah got it, thanks for the explanation!